terra-jupyter-python/Dockerfile

```diff
@@ -23,8 +23,8 @@ RUN pip3 -V \
  && pip3 install numpy==1.15.2 \
  && pip3 install py4j==0.10.7 \
  && python3 -mpip install matplotlib==3.0.0 \
- && pip3 install pandas==0.23.4 \
- && pip3 install pandas-gbq==0.11.0 \
+ && pip3 install pandas==0.25.3 \
+ && pip3 install pandas-gbq==0.12.0 \
```

Comment:
Should we also update `google-cloud-biquery` to `1.23.0` (see line 31)?

https://pypi.org/project/google-cloud-bigquery/1.23.0/

It seems like a few of our `google-cloud-*` libraries are out of date..

Reply:
be very careful with this... I attempted to do this while making the python image and upgrading any of the google library versions leads to nigh-unresolvable dependency hell